### PR TITLE
chore: v6.7.0 version bump

### PR DESCRIPTION
**Pull request by pomegranited**
**TL;DR -** Prepare release version

**What changed?**

- version now v6.7.0

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] <strike>Translations and JS/SASS compiled
- [x] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pomegranited! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by pomegranited** [comment]
Hi @mphilbrick211 @openedx/content-aurora
Could somebody help me out with this version bump? We're trying to get these updates into Redwood, which is cutting soon. Thanks!

**Comment by leangseu-edx** [comment]
@pomegranited `6.7.0` had already been release. https://github.com/openedx/edx-ora2/releases/tag/6.7.0

Do you think this pr is still needed?

**Comment by itsjeyd** [comment]
@mphilbrick211 

> I don't think Aurora is reviewing pull requests, ...

Per https://github.com/openedx/edx-ora2/blob/master/catalog-info.yaml#L16, Aurora maintains this repo. The file was created last month (by @jristau1984), so the info that it contains is probably up-to-date?

The main thing that's changed for this repo, I think, is that we should be pinging the Aurora team via @openedx/2u-aurora, and stop using the legacy @openedx/content-aurora handle.

Also, it's possible that the settings for this repo need some additional updates -- I tried requesting a review from `@openedx/2u-aurora` and it didn't work (auto-complete didn't turn up any results).

**Comment by openedx-webhooks** [comment]
@pomegranited Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-ora2&entry.867055334=https://github.com/openedx/edx-ora2/pull/2208&entry.1484655318=https://github.com/pomegranited&entry.752974735=2024-04-19+06:11&entry.1917517419=2024-04-30+02:59&entry.2133058324=No) so we can improve your experience in the future.

**Comment by pomegranited** [comment]
@leangseu-edx 
> 6.7.0 had already been release. https://github.com/openedx/edx-ora2/releases/tag/6.7.0 Do you think this pr is still needed?

Oh good, yep, that's all I needed. Thank you!

@itsjeyd @mphilbrick211 

> The main thing that's changed for this repo, I think, is that we should be pinging the Aurora team via @openedx/2u-aurora, and stop using the legacy @openedx/content-aurora handle.

Noted, thank you :)

> Also, it's possible that the settings for this repo need some additional updates -- I tried requesting a review from @openedx/2u-aurora and it didn't work (auto-complete didn't turn up any results).

I'm able to use the 2u handle ok? Not sure why it doesn't work for you.

**Comment by itsjeyd** [comment]
@pomegranited It works for me when I try to use it in a comment, but not in the context of the reviewers field:

image

Is it working everywhere for you?

**Comment by pomegranited** [comment]
@itsjeyd Ah.. no, I can't request a review from 2u-aurora either. From [github's docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review):

> Organization members with write access can also assign a pull request review to any person or team with read access to a repository.

Maybe 2u-aurora doesn't have read access to this repo? I'm unable to see the repo settings, so can't find out for myself.

@mphilbrick211 can you help?

**Comment by mphilbrick211** [comment]
@pomegranited @itsjeyd I'm going to check on this, as I'm not sure Aurora is still reviewing. If they are, I'll confirm if they need access.
